Dockerfile.windows

```diff
@@ -1,87 +1,163 @@
-# This file describes the standard way to build Docker, using a docker container on Windows
-# Server 2016
+# escape=`
+
+# -----------------------------------------------------------------------------------------
+# This file describes the standard way to build Docker in a container on Windows
+# Server 2016 or Windows 10.
+#
+# Maintainer: @jhowardmsft
+# -----------------------------------------------------------------------------------------
+
+
+# Prerequisites:
+# --------------
+#
+# 1. Windows 10 or Windows Server 2016 with all Windows updates applied. Pre-release
+#    versions of Windows are not supported (eg Windows Server 2016 TP5). The build
+#    number must be at least 14393. This can be confirmed, for example, by running
+#    the following from an elevated PowerShell prompt - this sample output is from a 
+#    fully up to date machine as at late October 2016:
 #
+#    >> PS C:\> $(gin).WindowsBuildLabEx
+#    >> 14393.321.amd64fre.rs1_release_inmarket.161004-2338
+#
+# 2. Git for Windows (or another git client) must be installed. https://git-scm.com/download/win.
+#
+# 3. The machine must be configured to run containers. For example, by following
+#    the quick start guidance at https://msdn.microsoft.com/en-us/virtualization/windowscontainers/quick_start/quick_start.
```

Comment:
The Docker Labs docs are more complete at this point, I think: https://github.com/docker/labs/blob/master/windows/windows-containers/Setup.md